app/subscriber/src/components/content-list/ContentList.tsx

```diff
@@ -135,11 +134,9 @@ export const ContentList: React.FC<IContentListProps> = ({
       array = [];
     }
     if (array.length !== selected?.length) {
-      array = _.uniq([...array, ...selected.map((i) => i.id)]);
-      localStorage.setItem('selected', JSON.stringify(array));
```

Comment:
This was introduced to initialize the page with the prior selected content when refreshing the page.  However, this breaks things when removing selected content or adding to report/folder.